src/modules/wishlist/types/wishlist.type.ts

```diff
@@ -0,0 +1,19 @@
+import { type Accommodation } from '#/modules/accommodations/types/accommodation.type';
+
+export interface WishlistItem {
```

Comment:
Please move the type to shared level. The modules should not connect with each other